### PR TITLE
Blog: ZAP Updates - May 2023

### PR DESCRIPTION
**Pull request by ricekot**
<details>
<summary>Screenshot (click to expand)</summary>

image

</details>

### CODE REVIEW

**Comment by kingthorin** [comment]
Auth Helper x3 versions, are the changes combine?

**Review by kingthorin — APPROVED** [review]
LGTM

**Review by psiinon — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by kingthorin — COMMENTED** [review]
LGTM

File: site/content/blog/2023-06-01-zap-updates-may-2023/index.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+[zaproxy](https://github.com/zaproxy/zaproxy/pulse/monthly)  
+Excluding merges, 6 authors have pushed 25 commits to main and 25 commits to all branches. On main, 71 files have changed and there have been 2,227 additions and 1,069 deletions. 
+
+[zap-extensions](https://github.com/zaproxy/zap-extensions/pulse/monthly)  
+Excluding merges, 9 authors have pushed 66 commits to main and 66 commits to all branches. On main, 1,641 files have changed and there have been 84,912 additions and 83,622 deletions. 
+
+## Released add-ons - Full Changelog
+In May 2023, we released updated versions of 19 add-ons:
+
+##### Active scanner rules (beta) (v46)
```
[the remark this answers is not in the document]
Comment (kingthorin) [reply site/content/blog/2023-06-01-zap-updates-may-2023/index.md:98]: Oops I was too late. Food for thought for the next one :grinning:

**Pull request merged**